lib/modules/manager/sbt/extract.ts

```diff
@@ -32,10 +37,41 @@ interface Ctx {
   depType?: string;
   useScalaVersion?: boolean;
   variableName?: string;
+
+  sbtVersion?: string;
 }
 
 const scala = lang.createLang('scala');
 
+const sbtVersionMatch = q
+  .sym<Ctx>('sbt')
+  .op('.')
+  .sym('version')
+  .op('=')
+  .num((ctx, { value: sbtMajor }) => ({ ...ctx, sbtVersion: sbtMajor }))
```

Comment:
The 3 `num` are kinda ugly but that's the way the parser reads `1.6.0`, if there's any better way I'll be happy to change this!

Reply:
@zharinov i think we should use a simple regex parser for the properties file

Reply:
I agree that a simple regex clause would be good enough. It seems wrong to me to apply a parser setup for Scala onto a properties file that is not in Scala.

Reply:
Yes, it's better to dispatch on `packageFile` parameter in `extractPackageFile()` function and choose appropriate implementation.

It looks reasonable to just match the entire `.properties` file content on `sbt.version = 1.2.3" expression.

Reply:
When you say use simple regex parser, do you have pointer where such regex parser is already used?

Reply:
just add a conditional here
https://github.com/renovatebot/renovate/blob/a340830ee6de5cea806f0db27fce379382d3c07e/lib/modules/manager/sbt/extract.ts#L292

if packageFile is a properties file, use regex to extract the version and return the SBT dependency